tests/test_app.py

```diff
@@ -135,11 +301,29 @@ def test_app_init_with_invalid_options(self, options):
         with pytest.raises(ValueError):
             firebase_admin.initialize_app(CREDENTIAL, options=options)
 
+    @pytest.mark.parametrize('bad_file_name', bad_config_file)
+    def test_default_app_init_with_bad_config_from_env(self, bad_file_name):
+        config_file_old = os.environ.get(CONFIG_JSON)
+        os.environ[CONFIG_JSON] = testutils.resource_filename(bad_file_name)
+        try:
+            with pytest.raises(ValueError):
+                firebase_admin.initialize_app(CREDENTIAL)
+        finally:
+            if config_file_old:
```

Comment:
I think you should create a fixture, that sets a given value/file as the FIREBASE_CONFIG variable, and then reverts it back after the test. Then inject that fixture to any tests that is potentially affected by this change. You can probably get rid of that session scoped/autouse fixture that way.

Reply:
This is exactly what I am doing in the OptionsTest class